anthos-onprem-terraform/avmw_user_cluster_metallb/README.md

```diff
@@ -58,20 +58,31 @@ have changed directory to where this samples is:
 
 ### Upgrade the user cluster with terraform
 
-Before upgrading the user cluster, please make sure the admin cluster platform
-controller has been upgraded to the target version. The steps to upgrade the
-admin cluster platform controller is listed in the
-[public documentation](https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/upgrade-on-prem-api#available_versions_for_upgrades).
+Before upgrading the user cluster, please make sure the admin cluster has been 
```

Comment:
_Please add the following description at the beginning:_

Use the same terraform script to upgrade the user cluster, by simply changing the version to the new version. Note that this script can be used for upgrades only if you had created the user cluster using this script. When you run the script with the updated version, the `terraform.tfstate` created during the first run of the script is compared to recognize the change.

Before upgrading the user cluster, please.....

Reply:
Added